returnn/datasets/distrib_files.py

```diff
@@ -255,17 +269,21 @@ def init_seq_order(self, epoch: Optional[int] = None, seq_list=None, seq_order=N
             if full_epoch_0idx_ in self._files_order_cache:
                 continue
             if self.seq_ordering == "default":
-                files_order_flat = self.files
+                files = self.files
             elif self.seq_ordering == "random":
-                rnd_seed = self._get_random_seed_for_epoch(full_epoch_0idx_ * self.partition_epoch + 1)
-                random_generator = numpy.random.RandomState(rnd_seed)
-                files_order_flat = list(self.files)
-                random_generator.shuffle(files_order_flat)
+                random_generator = numpy.random.RandomState(
+                    self._get_random_seed_for_epoch(full_epoch_0idx_ * self.partition_epoch + 1)
+                )
+                files = list(self.files)
+                random_generator.shuffle(files)
```

Comment:
It's actually here exactly the same code as before, only that you changed the var name. Maybe just keep the original code, so the diff looks cleaner?